sema/ro/creator/rocreator.py

```
@@ -0,0 +1,56 @@
+import logging
+
+from sema.commons.service import ServiceBase, ServiceResult, Trace
+
+log = logging.getLogger(__name__)
+
+
+class RocResult(ServiceResult):
+    """Result of the Roc service"""
+
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)